android/src/com/mapswithme/util/sharing/TargetUtils.java

```diff
@@ -16,18 +13,9 @@
 
   private TargetUtils() {}
 
-  public static void fillSmsIntent(Activity activity, Intent smsIntent, String body)
+  public static void fillSmsIntent(Intent smsIntent, String body)
   {
     smsIntent.setData(Uri.parse(URI_STRING_SMS));
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT)
-    {
-      final String defaultSms = Telephony.Sms.getDefaultSmsPackage(activity);
```

Comment:
А потому что без неё по факту лучше работает)

> On Mar 28, 2016, at 3:52 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/util/sharing/TargetUtils.java <https://github.com/mapsme/omim/pull/2568#discussion_r57568394>:
> 
> >    {
> >      smsIntent.setData(Uri.parse(URI_STRING_SMS));
> > -    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT)
> > -    {
> > -      final String defaultSms = Telephony.Sms.getDefaultSmsPackage(activity);
> А почему эта ветка вырезана?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2568/files/f2dc591f49716076ca1919ceee98db764511cb08#r57568394>